src/tv2-constants/enums.ts

```diff
@@ -192,7 +192,8 @@ export enum AbstractLLayer {
 export enum SwitcherMixEffectLLayer {
 	Program = 'me_program',
 	Clean = 'me_clean',
-	CleanUSKEffect = 'clean_usk_effect',
+	CleanUskFull = 'clean_usk_full',
```

Comment:
I know you followed the casing that' allready present in the Enum, but I would like us to write our enums in SNAKE_CASE. I.e:
```
export enum SwithcerMixEffectLLayer {
  PROGRAM = 'me_program',
  CLEAN_USK_FULL = 'clean_usk_full',
  ...
}
```
It makes the Enums clearly stand out when used elsewhere in the code :)

Reply:
I really like this too, i wish we had it everywhere :-)

Reply:
Done for this and some other enums, also removing some redundant words like  `SwitcherAuxLLayer.AuxProgram` -> `SwitcherAuxLLayer.PROGRAM`